pom.xml

```diff
@@ -167,14 +167,19 @@
                 </includes>
                 <excludes>
                   <exclude>**/plugin.xml</exclude>
-                  <exclude>pom.xml</exclude>
+                  <exclude>**/pom.xml</exclude>
+                  <exclude>**/dependency-reduced-pom.xml</exclude>
+                  <exclude>**/gradle.xml</exclude>
+                  <exclude>**/target/</exclude>
```

Comment:
Why are we adding these excludes?

Reply:
Good question. It's not to format those XML files. The ones in target don't get stored in Git.

Reply:
`pom.xml` was already there. I just fixed the path pattern.

Reply:
The dependency reduced pom doesn't get stored in Git either. I guess we could format these if we want?

Reply:
I don't have a particular opinion here 🙂 but why do we need to skip formatting?  Does it lead to a bad format?  Or does it make Spotless take too long to run?

Reply:
With the files not stored in Git, right, it's to save time; the generated XML files can be quite large. With the others, let me try it and see how it looks. I cannot remember now why we originally skipped `pom.xml`, for example.

Reply:
Looks like at least `pom.xml` was excluded here: https://github.com/wala/ML/pull/61/commits/d74b32b41a2be6fe86cf736294476ccf1f460dfb. The comment says:
> Ignore pom.xml in XML formatting.
We have another formatter for that.

I am guessing that m2e is doing it's own formatting each time the file is edited using m2e. Other than that, I can't recall what other formatter we could be using ...

Reply:
NM, it's here: https://github.com/wala/ML/blob/c40ccba918a524bb50418db1888a6c609b48b4a0/pom.xml#L186. 

It's actually already being done by spotless.